lib/main.dart

```diff
@@ -105,3 +111,108 @@ class _NavBarState extends State<NavBar>
     );
   }
 }
+
+class PublicationsPage extends StatelessWidget {
```

Comment:
Would you be able to move the PublicationsPage into its own file in a pages/ directory?

Reply:
Alright, I've made those changes.